From f654fbc314694cce5ec54d8a4e68add44d01abb4 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Thu, 5 Jul 2018 11:32:14 +0800
Subject: [PATCH 200/205] drm: compile drm_vm.c omap needed

drm_vm is needed by pvrsrvkm, so we select by Kconfig

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/omapdrm/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/omapdrm/Kconfig b/drivers/gpu/drm/omapdrm/Kconfig
index b3d08c5..5291ddf 100644
--- a/drivers/gpu/drm/omapdrm/Kconfig
+++ b/drivers/gpu/drm/omapdrm/Kconfig
@@ -3,6 +3,7 @@ config DRM_OMAP
 	depends on DRM
 	depends on ARCH_OMAP2PLUS || ARCH_MULTIPLATFORM
 	select OMAP2_DSS
+	select DRM_VM
 	select DRM_KMS_HELPER
 	default n
 	help
-- 
1.7.9.5

